app/src/androidTest/java/org/mozilla/fenix/syncintegration/Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+fxapom = "*"
+mozdownload = "*"
+mozinstall = "*"
+mozprofile = "*"
+mozrunner = "*"
+mozversion = "*"
+pytest = "*"
+pytest-fxa = "*"
+pytest-html = "*"
+pytest-metadata = "*"
+requests = "*"
```

Comment:
do you not want to pin down these version numbers for stability?

Reply:
We have had this for the iOS tests for long time and did not face any stability issues...but if you prefer I change that, I can look into that.